.travis.yml

```diff
@@ -4,7 +4,7 @@ sudo: false
 language: java
 jdk: openjdk11
 cache: { directories: [ ".gradle", "~/.gradle/caches" ] }
-install: skip
+install: skip travis_retry
```

Comment:
AFAIK "skip" is just a placeholder term to skip the install task, so it doesn't make sense to keep it here

Reply:
+From the documentation it seems that this is a built-in command no need to run it inside the install step

Reply:
True enough @RoiEXLab 
I must have been tired or something when doing this, I was thinking this was a list of dependencies to install and that 'skip' was one of them somewhere..  :grin: 

Yeah, 'install' block is a script to run, makes sense for this to have error'ed out.